olp-cpp-sdk-core/include/olp/core/client/OlpClient.h

```diff
@@ -65,7 +65,17 @@ class CORE_API OlpClient {
    * @brief Set the client settings.
    * @param settings Client settings.
    */
-  void SetSettings(const OlpClientSettings& settings);
+  inline void SetSettings(const OlpClientSettings& settings) {
```

Comment:
Why this code is changed?

Reply:
to align all calls of OlpClient::SetSettings to new approach without capturing the task scheduler